tests/e2e/workloads/pgsql/test_pgsql_ocp_shutdown.py

```diff
@@ -0,0 +1,88 @@
+import logging
```

Comment:
We are doing recover post shutdown :)
May be we can change the script name to mention it